azure-pipelines.yml

```diff
@@ -1,27 +1,23 @@
-# https://aka.ms/yaml
-variables:
-  # Current tooling indexes directly into os.environ and will
-  # fail without this defined
-  # TODO: Implement specific PR logic and then fix up tooling to be okay
-  # without this environment variable set
-  travis_event_type: ''
-
+# Schema docs at https://aka.ms/yaml
+trigger:
+  - master
 
 jobs:
   - job: precheck_concurrent
     # This job schedules long-running tasks concurrently with the rest of the
     # build pipeline ensuring that later stages don't block on them
+    # We'll merge it back into the `precheck` stage once Pipelines has caching.
     pool:
       vmImage: 'Ubuntu 16.04'
     strategy:
       matrix:
         check-whole-repo-tests:
           TASK: check-whole-repo-tests
     steps:
-    - script: >
-        sudo apt-get install
-        libreadline-dev libsqlite3-dev
+    - script: "sudo apt-get install libreadline-dev libsqlite3-dev"
```

Comment:
Nit pick, why is this string quoted when others are not?

Reply:
...I think I was imitating a different example config.  Will take the quotes out.